### ISSUE TITLE
Document the platforms supported

### ISSUE THREAD

**Issue by mike-myers-tob** [trailofbits/it-depends#60]
If I recall, C/C++ dependency enumeration is only supported on Linux. Might want to indicate this on the README.

**Comment by ojuschugh1** [comment]
Hi @mike-myers-tob, I hope you are doing great. Is this issue still unresolved? I wanted to work on this issue if it is not solved.

Thanks and Regards

**Comment by mike-myers-tob** [comment]
Yes, it appears that still nobody has changed the README to indicate how different features are available only on certian platforms. If you wish to open a pull request to address this issue, you are welcome to do so!

**Comment by ojuschugh1** [comment]
Thank you for your response. I will test the program on both Windows and Linux (Ubuntu) platforms to identify any issues and assess the supported features. Am I understanding this issue correctly?

**Comment by ojuschugh1** [comment]
Hey @mike-myers-tob , please could you confirm this.

**Comment by mike-myers-tob** [comment]
Yes I don't think you need any more than this. If you cannot confirm that a claimed feature works on a particular platform, you can document that. It may be known thing or it may be a bug, but a maintainer here will have to review that when there is a pull request for the update to the documentation.

**Comment by ojuschugh1** [comment]
Thanks @mike-myers-tob  Sir, for your reply. I will do that !!
